apps/publisher/extensions/app/publisher-apis/apis/taxonomy.jag

```diff
@@ -29,53 +29,52 @@ require('/modules/publisher.js').exec(function (ctx) {
     var utils = require('utils');
     var ref = utils.request;
 
-    switch (method) {
-        case 'GET':// GET endpoints
+        switch (method) {
+            case 'GET':// GET endpoints
 
-            if (uriMatcher.match(TAXA)) {
-                acceptHeader = req.getHeader('Accept') || constants.ACCEPT_ALL;
-                jsonHeaderAccept = acceptHeader.indexOf('application/json');
-                try {
-                    if (jsonHeaderAccept >= 0 || acceptHeader == constants.ACCEPT_ALL) {
-                        var taxaPathList;
-                        var options = ref.getQueryOptions(req.getQueryString());
-                        if (options['terms']) {
-                            taxaPathList = taxa.taxonomy.getNodesList(options['terms'],
-                                    ( options['startnode']) ? options['startnode'] : 0,
-                                    (options['endnode']) ? options['endnode'] : 5);
+                if (uriMatcher.match(TAXA)) {
+                    acceptHeader = req.getHeader('Accept') || constants.ACCEPT_ALL;
+                    jsonHeaderAccept = acceptHeader.indexOf('application/json');
```

Comment:
Thank you Sameera for suggestion,
This has been fixed with this PR https://github.com/wso2/carbon-store/pull/834